content/news/playtest-20150102.md

```diff
@@ -0,0 +1,34 @@
+We wish everyone a happy new year and prepared a new playtest which builds upon our [Halloween release](/news/release-20141029/). The main focus was to improve stability and performance as well as implement unique features of Dune 2000 that distinguish it from the other mods.
+
+
+These playtest builds provide a snapshot of our current development for feedback by our testers. You can help shape the future of OpenRA by testing these builds and [reporting bugs](http://bugs.openra.net) or simply letting the [community](/community/) know what you think.
+
+Some highlights in this development build are:
+
+   - Added an often requested lobby option for disabling the "Short Game" mode.
+   - Added a difficulty selector for single-player missions and FMV briefings.
+   - Added hardware cursors support for smoother mouse movements.
+   - Added new missions: GDI 03, Soviet 01 and Survival 01.
+   - Fixed Fort Lonestar and the Dropzone minigames not starting.
+
+See the [changelog](http://changelog.openra.net) for more details on the playtest changes and bugfixes.
+
+The playtest-20150102 build is available from the [download](/download/) page. We handed over packaging in regards of RPM for Fedora and openSUSE as well as Arch Linux completely to the Linux community as they produce better results than our build servers. Note to packagers: we switched dependency management to [NuGet](http://www.nuget.org/).
```

Comment:
I think we can get away with being a bit more (but hopefully not too) detailed here, maybe something like: "We have put a lot of work into streamlining our development and release process, which included a shake-up of our Linux packages.  We now provide distro-specific RPM and Arch Linux packages via the [openSUSE Build Service](https://software.opensuse.org/download.html?project=games:openra&package=openra), and recommend that most users install OpenRA from their main distribution repositories.  We have also fixed the "bad quality" package warning that occurs when installing our Ubuntu package."